release-team/shadows.md

```diff
@@ -10,7 +10,7 @@ The following guide is for both Release role leads and shadows so that everyone
 
 In the last week of the prior release cycle, or the first week of the new one, the Emeritus Advisor and Release Lead will post a sign-up form for people who want to be Shadows.  Potential Shadows should fill out this form, which has a number of questions about their interests, availability, and experience; this is to judge who will be the most suitable and balance the team.  
 
-Please read the [Role Handbooks](https://github.com/kubernetes/sig-release/tree/master/release-team) for those roles before filling out the form.  Some handbooks will have a specific Shadow section that addresses what background Shadows are expected to have and what tasks they do.
```

Comment:
I think this link may be intentional since it lands you on the README for a summary of release team roles

Reply:
I might be wrong here, but IMO either anchor or link is misleading, do you think it would be good idea to change anchor from "Role Handbooks" to something else as an alternative?

Reply:
@ParvJain sorry for the delay here! Looking back here, I think your update is appropriate :)